### DISCUSSION TITLE
Old value store on editor.getHtml

### DISCUSSION THREAD

**Discussion by cfbadr** [GrapesJS/grapesjs#4639] [Q&A]
Hello, I need to store the html  I updated on the editor.

So when user click on my save button,  I trigger my function `saveNewTemplate()`

Inside this function I'm calling `editor.getHtml();`, but the value  returned is not always the good one, sometime yes, and sometime  no, it return the old values.

Do I have to put a setTimeout or something to be sure to have the last values stored  ?
Do I have to init  grapeJs everytime the page is rendered ?
Or maybe it's coming from something else ?

Thanks

**Comment by hyuuks** [comment]
im sure your question related to grapesjs.com/docs/api/editor.html#store
